haikubot/commands/commands_parser.py

```diff
@@ -245,9 +250,9 @@ def _wordcloud(self, command, channel):
             if not is_sprint:
                 haikus = self.store.get_all_haiku()
             else:
-                haikus = self.store.get_all_haiku_weeks(4)
+                haikus = self.store.get_all_haiku_weeks(3)
```

Comment:
I corrected the number of weeks to to match a (our) sprint. Maybe it should be set in the config?

Reply:
#21